component/loki/process/internal/metric/counters.go

```diff
@@ -63,7 +63,7 @@ func (c *CounterConfig) UnmarshalRiver(f func(v interface{}) error) error {
 	return nil
 }
 
-// Counters is a vec tor of counters for a each log stream.
+// Counters is a vector of counters for a log stream.
```

Comment:
@tpaschalis I am not 100% certain. Did you mean `a log stream` or `each log stream`?

Reply:
The proposed change sounds okay (eg. for a given log stream)